spec/jobs/clock/refresh_wallets_credits_job_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 require 'rails_helper'
 
-describe Clock::RefreshWalletsCreditsJob, job: true do
+describe Clock::RefreshWalletsOngoingBalanceJob, job: true do
```

Comment:
RSpec/FilePath: Spec path should end with `clock/refresh_wallets_ongoing_balance_job*_spec.rb`. (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FilePath)